### DISCUSSION TITLE
function(){var m,aa=function(a){var b=0;return function(){return b<a.length?

### DISCUSSION THREAD

**Discussion by eumaei** [community/community#86775] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

function(){var m,aa=function(a){var b=0;return function(){return b<a.length?

**Comment by hangshan-boop** [comment]
#include <stdio.h>
int main(int a, int b)
{
return (a<b)?a:b ;          //if a<b,returned a,else if b
}

**Reply by hangshan-boop** [reply]
This can obtain the length.

**Comment by anuragjain-git** [comment]
You have to write something to be executed if condition is true or false

### Ternary Operator
`condition ? expressionIfTrue : expressionIfFalse;`

```javascript
function() {
    var m, aa = function(a) {
        var b = 0;
        return function() {
            return b < a.length ? /* if true */ : /* if false */;
        };
    };
    return aa;
}
```

- You can simplify the code aswell by simplifying inner function to be an arrow function with implicit returns

```javascript
function example() {
    const b = 0;
    return a => () => b < a.length ? /*if statement*/ : */*else statement*/;
}

**Comment by hangshan-boop** [comment]
`#include <stdio.h>`
` int main (int a, int b)
{
return (a<b)?a:b ;
}`
